api/v1beta1/restore_types.go

```diff
@@ -25,18 +25,18 @@ type RestoreSpec struct {
 	// VeleroManagedClustersBackupName is the name of the velero back-up used to restore managed clusters.
 	// Is optional, if not supplied, the backup name will be selected
 	// from the available backups in the current namespace
-	// +kubebuilder:validation:Optional
-	VeleroManagedClustersBackupName *string `json:"veleroManagedClustersBackupName,omitempty"`
+	// +kubebuilder:validation:Required
+	VeleroManagedClustersBackupName *string `json:"veleroManagedClustersBackupName"`
 	// VeleroResourcesBackupName is the name of the velero back-up used to restore resources.
 	// Is optional, if not supplied, the backup name will be selected
 	// from the available backups in the current namespace
-	// +kubebuilder:validation:Optional
-	VeleroResourcesBackupName *string `json:"veleroResourcesBackupName,omitempty"`
+	// +kubebuilder:validation:Required
+	VeleroResourcesBackupName *string `json:"veleroResourcesBackupName"`
 	// VeleroCredentialsBackupName is the name of the velero back-up used to restore credentials.
 	// Is optional, if not supplied, the backup name will be selected
```

Comment:
This needs to change to : 
Required, valid values are latest, skip or _backup_name_
If value is set to latest, the latest backup is used, skip will not restore this type of backup; _backup_name_ points to the name of the backup to be restored